addons/web/static/src/core/popover/popover.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="web.PopoverWowl" owl="1">
         <div role="tooltip" class="o_popover popover mw-100" t-att-class="props.class" t-ref="ref">
             <t t-slot="default" />
-            <div class="popover-arrow"/>
+            <div t-if="!props.noReposition" class="popover-arrow"/>
```

Comment:
Not a fan of "negative meaning variables".

Could we go for "fixedPosition" or something similar instead?

Reply:
ok :)